lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -56,6 +65,15 @@ defmodule Kernel.ParallelCompiler do
     spawn_compilers(files, path, options)
   end
 
+  @doc """
+  Requires the given files in parallel.
+  Read `files/2` for more information.
+  """
+  def require_files(files, options \\ []) do
```

Comment:
Let's just change the ParallelCompiler then. If the issue is with testing, then we may even figure out a way to make `mix test` use the ParallelCompiler instead.